projects/1.18/assets/macaws-roofs/mcwroofs/lang/zh_cn.json

```
@@ -117,6 +124,13 @@
   "block.mcwroofs.warped_planks_steep_roof": "诡异木木板陡峭屋顶底端",

```

```suggestion
  "block.mcwroofs.warped_planks_steep_roof": "诡异木板陡峭屋顶底端",
```
按照现行[译名标准化表](https://minecraft.fandom.com/zh/wiki/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96)`Wrapped Planks`应为`诡异木板`
同样地`Mangrove Planks`应为`红树木板`

---

若要更改，至L133为止的`木木板`应作统一修改。

---

这些`木板`作为木制品名称的一部分出现，类似`铁轨`中的`铁`

---

审阅的时候也看到了这个木木，但没提
这个问题我一直都在困扰我，按[原版译名](https://minecraft.fandom.com/zh/wiki/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96)，有`深色橡木木板`，又有`诡异木板`。前者可以按Chuijk的说法解释，但后者可以看作是偏正短语，也可以看作是`诡异木木板`的“缩减版”*。

原版只有橡木和深色橡木是`木木板`。

* 我不知道这个语言现象的标准说法，就像`文化部部长`可以讲成`文化部长`，而后者也是官方说法。

---

> 审阅的时候看到了这个木木，但没提。 这个问题我一直都在困扰我，按[原版译名](https://minecraft.fandom.com/zh/wiki/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96)，有`深色橡木木板`，又有`诡异木板`。前者可以按Chuijk的说法解释，但后者可以看作是偏正短语，也可以看作是`诡异木木板`的“缩减版”*。
> 
> 原版只有橡木和深色橡木是`木木板`。
> 
> * 我不知道这个语言现象的标准说法，就像`文化部部长`可以讲成`文化部长`，而后者也是官方说法。

其实我也支持把二字树名后面的`木`去掉
一开始的考量确实就和我上面说的一样，主要是标准化木制品的名称（更进一步其实是为了减少兼容性模组名称生成的奇妙程度（））
不过这么做不符合原版的命名规范，那还是恢复成原版的命名方式吧

---

我感觉这种译名有种“逃避单音节组分”的感觉......毕竟“橡木板”听着还是挺别扭的
至于深色橡木估计可以认为是与橡木的名称统一

~也有用格式符做名称适配的问题......据说bugjump曾经在死亡信息上这么试过一次，然后被非英语族给骂了~

---

> 我感觉这种译名有种“逃避单音节组分”的感觉......毕竟“橡木板”听着还是挺别扭的 至于深色橡木估计可以认为是与橡木的名称统一
> 
> ~也有用格式符做名称适配的问题......据说bugjump曾经在死亡信息上这么试过一次，然后被非英语族给骂了~

这边的主要问题就是everycomp（）
假如是`%s板`则单字木材字数不够，`%s木板`则多字木材都会多出一个`木`
属于是非常头大了

---
目前我的处理方式是
* 单字树名（橡木、柏木、榉木）采用原版`橡木木板`、`橡树树叶`、`橡木台阶`
* 普通多字（白桦木、棕榈木、金合欢木）采用原版`白桦木板`、`白桦树叶`、`白桦木台阶`
* 特殊多字（主要是存在对应单字树名的多字树名）采用原版`深色橡木木板`、`深色橡树树叶`、`深色橡木台阶`
* 特殊名称（竹），自成一派（），原版`竹板`、`竹门`、`竹马赛克`

没法准确裁定的是特殊多字（如Biome Makeover中的沼泽柏木/`Swamp Cypress`，若干模组中的彩虹桉木/`Rainbow Eucalyptus`），这些名称在现实生活中有对应单字树名，但其本身也可作为单独的名字

我是有这么几种做法
* 全部不管，视为普通多字（惯用做法）
* 全部不管，视为特殊多字
* 原文多个词组成的，视为特殊多字；原文单个词组成的，视为普通多字
    * （沼泽柏木/`Swamp Cypress`，蓝花楹木/`Jacaranda`）
* 幻想种/现实中非分类单种，视为特殊多字；其余视为普通多字
    * （沼泽柏木/`Swamp Cypress`，彩虹桉木/`Rainbow Eucalyptus`（_Eucalyptus deglupta_））

完全不会裁定（）
比如前几天的日本红枫/`Japanese Maple`就出过这种问题（#2704，_Acer palmatum_，鸡爪槭）